pkg/gofr/handler.go

```diff
@@ -59,13 +61,24 @@ func (h handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	done := make(chan struct{})
+	panicked := make(chan struct{})
 
 	var (
 		result interface{}
 		err    error
 	)
 
 	go func() {
+		defer func() {
+			re := recover()
+			if re != nil {
+				close(panicked)
+				h.container.Error(panicLog{
+					Error:      fmt.Sprintln(re),
+					StackTrace: string(debug.Stack()),
+				})
+			}
+		}()
```

Comment:
we can also consider creating a `panicRecovery` function, similar to https://github.com/gofr-dev/gofr/blob/development/pkg/gofr/http/middleware/logger.go#L127

Reply:
ok , i will do it now

Reply:
I test it before do this , I also want to call a recover func  but only direct call recover work .

Reply:
All Done @srijan-27